doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -261,7 +261,11 @@ Debug samples
 DECT NR+ samples
 ----------------
 
-|no_changes_yet_note|
+* :ref:`dect_shell_application` sample:
+
+  * A brief mac level sample on top of DECT PHY interface. New commands to create a periodic cluster beacon, scan for it, associate/disassociate a PT/client, and send data to a FT/beacon random access window. Note: this is just a sample/initials of a mac implementation and thus not a full MAC implementation and not fully compliant with dect nr+ mac spec (ETSI TS 103 636-4).
+  * dect rssi_scan: In addition to default high/low rssi_scan: option for MAC spec like rssi_scan: busy/possible/free subslot count based RSSI scan.
+  * dect rx: possibility to iterate all channels and to enable RX filter.
```

Comment:
```suggestion
  * Added:

    * A brief MAC-level sample on top of DECT PHY interface with new commands to create a periodic cluster beacon, scan for it, associate or disassociate a PT/client, and send data to a FT/beacon random access window.
      This is s not a full MAC implementation and not fully compliant with DECT NR+ MAC specification (ETSI TS 103 636-4).
  * ``dect rssi_scan``: in addition to the default high/low ``rssi_scan`` option. 
  * ``dect rx``: possibility to iterate all channels and to enable RX filter.
```
A bit too complex to my taste. Does this still work? You could also add a link to the ETSI spec (links.txt file).